benchmark_runner/common/template_operations/templates/hammerdb/hammerdb_data_template.yaml

```diff
@@ -38,21 +38,21 @@ template_data:
       requests_memory: 10Mi
   extra:
     mariadb:
-      db_image: centos8-mariadb103-container-disk:latest
+      db_image: centos-stream8-mariadb103-container-disk:latest
```

Comment:
All of these should use the quay.io/cloud-bulldozer image, not the default docker.io ones.

Reply:
Full path [here](https://github.com/redhat-performance/benchmark-runner/blob/b7858ee7f63aa56ddf13e582beabf5d4d8c2b6bf/benchmark_runner/common/template_operations/templates/hammerdb/internal_data/hammerdb_vm_template.yaml#L80)

Reply:
Yep, OK.